cmd/soroban-cli/src/invoke.rs

```diff
@@ -49,6 +50,9 @@ pub struct Cmd {
     /// Argument to pass to the function (base64-encoded xdr)
     #[clap(long = "arg-xdr", value_name = "arg-xdr", multiple = true)]
     args_xdr: Vec<String>,
+    /// File containing JSON Argument to pass to the function
+    #[clap(long, multiple = true)]
```

Comment:
the e2e test fails because it passes --arg-file and the command is not found.  so the test works, maybe change
line 54:
`#[clap(long,` multiple = true)]`
to
`#[clap(long = "args-file", value_name="args-file", multiple = true)]`

Reply:
clap converts snake_case to kebab-case, so the rename isn't necessary. The issue was I passed `--arg-file` instead of `--args-file`. After finally setting up the docker server for testing locally it seems to pass now.

Reply:
Ah didn't notice that missing 's'.  It worked locally for me also!